### PR TITLE
build: bump buildifier to 6.4.0

### PR DESCRIPTION
**Pull request by chokobole**
# Description

This PR bumps up buildifier to 6.4.0 and runs buildifier

### CODE REVIEW

**Review by Insun35 — COMMENTED** [review]
Please fix the typo in your commit message: biuld -> build

**Review by Insun35 — APPROVED** [review]
LGTM

File: vendors/halo2/BUILD.bazel [hunk 1]
```diff
@@ -14,18 +14,18 @@ tachyon_rust_library(
     crate_features = FEATUERS,
```
[the remark this answers is not in the document]
Comment (chokobole) [reply vendors/halo2/BUILD.bazel:14]: Yeap i'll do

**Review by dongchangYoo — APPROVED** [review]
LGTM

**Pull request merged**